core/src/test/java/io/aiven/klaw/service/TopicControllerServiceTest.java

```diff
@@ -322,8 +552,8 @@ public void deleteTopicRequests() throws KlawException {
   }
 
   @Test
-  @Order(13)
-  public void approveTopicRequestsSuccess() throws KlawException {
+  @Order(23)
+  public void approveTopicRequestsSuccess1() throws KlawException {
```

Comment:
We should give it a proper name instead of `1` and `2`.

Reply:
Agree, updated.